task_sdk/tests/execution_time/test_task_runner.py

```diff
@@ -63,10 +87,15 @@ def test_recv_StartupDetails(self):
         assert decoder.request_socket.fileno() == w2.fileno()
 
 
-def test_parse(test_dags_dir: Path):
+def test_parse():
+    """Test that checks parsing of a basic dag."""
+    dag = get_inline_dag("super_basic", BaseOperator(task_id="a"))
+    agency = SpyAgency()
+    agency.spy_on(parse, call_fake=lambda what: mocked_parse(what, dag, "a"))
```

Comment:
KGB gives a fixtue called `spy_agency` so

```suggestion
def test_parse():
    """Test that checks parsing of a basic dag."""
    dag = get_inline_dag("super_basic", BaseOperator(task_id="a"))
    spy_agency.spy_on(parse, call_fake=lambda what: mocked_parse(what, dag, "a"))
```

Reply:
Oh yeah, simpler. Making this change